ggshield/cmd/auth/auth_config.py

```diff
@@ -0,0 +1,183 @@
+from typing import Optional, Union
+
+import click
+
+from ggshield.core.config import (
+    AuthConfig,
+    Config,
+    InstanceConfig,
+    UnknownInstanceError,
+)
+
+
+DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+FIELD_OPTIONS = ("default_token_lifetime",)
+
+
+@click.command()
+@click.pass_context
+def config_list_cmd(ctx: click.Context) -> int:
+    """
+    List all auth configurations. Example of output
+
+    [default]
+    default_token_lifetime: 14d
```

Comment:
it should be `14` as a number to match the auth config format. There's a mistake in the specs for this particular command.